applications/serial_lte_modem/doc/FOTA_AT_commands.rst

```diff
@@ -56,7 +56,6 @@ Syntax
 .. note::
 
    When doing full modem FOTA, an external flash is used to store the firmware.
```

Comment:
```suggestion
   When doing full modem FOTA, external flash is used to store the firmware.
```

Reply:
Fixed, sorry for several rounds.

Reply:
No worries. I blame the GitHub preview not displaying the change properly.